### PR TITLE
📝 New Blog - Announcing TinaCMS Version 2.0

### PR DESCRIPTION
**Pull request by JackDevAU**
### General Contributing:

- [ ] Have you followed the guidelines in our [Contributing document](https://github.com/tinacms/tina.io/blob/master/CONTRIBUTING.md)?
- [ ] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?

### All New Content Submissions: (To be confirmed by reviewer)

- [ ] Title is short & specific
- [ ] Headers are logically ordered & consistent
- [ ] Purpose of document is explained in the first paragraph
- [ ] Procedures are tested and work
- [ ] Any technical concepts are explained or linked to
- [ ] Document follows structure from templates
- [ ] All links work
- [ ] The spelling and grammar checker has been run
- [ ] Graphics and images are clear and useful
- [ ] Any prerequisites and next steps are defined.

### CODE REVIEW

File: content/blog/version-2.mdx
Comment (joshbermanssw) [inline content/blog/version-2.mdx:1]: should there be a space around the hyphen, ".... But that's not all—TinaCMS has ....."

File: content/blog/version-2.mdx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+---
+
+### Announcing TinaCMS Version 2.0: A New Era of Content Management
+
+We are thrilled to announce the launch of TinaCMS Version 2.0! This major update marks a significant milestone in our journey, bringing powerful new features and enhancements. But that's not all—TinaCMS has been making waves in the tech community, and we're excited to share some of our recent achievements and events.
+
+
+#### TinaCMS Joined the SSW Family
+
+We were very proud to announce that TinaCMS had officially become part of the SSW family. Acquired by SSW, an Australian software development company and long-time Tina customer, this acquisition marked a new chapter for TinaCMS. This partnership will allow us to further enhance our platform and continue delivering top-notch solutions to our community. Rest assured, there will be no disruptions to our service, and our commitment to innovation remains stronger than ever.
```
Comment (bradystroud) [inline content/blog/version-2.mdx:16]:
```suggestion
We were very proud to announce that TinaCMS had officially become part of the SSW family. Acquired by SSW, an Australian software consultancy and long-time Tina customer, this acquisition marked a new chapter for TinaCMS. This partnership will allow us to further enhance our platform and continue delivering top-notch solutions to our community. Rest assured, there will be no disruptions to our service, and our commitment to innovation remains stronger than ever.
```

File: content/blog/version-2.mdx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+---
+
+### Announcing TinaCMS Version 2.0: A New Era of Content Management
+
+We are thrilled to announce the launch of TinaCMS Version 2.0! This major update marks a significant milestone in our journey, bringing powerful new features and enhancements. But that's not all—TinaCMS has been making waves in the tech community, and we're excited to share some of our recent achievements and events.
+
+
+#### TinaCMS Joined the SSW Family
+
+We were very proud to announce that TinaCMS had officially become part of the SSW family. Acquired by SSW, an Australian software consultancy and long-time Tina customer, this acquisition marked a new chapter for TinaCMS. This partnership will allow us to further enhance our platform and continue delivering top-notch solutions to our community. Rest assured, there will be no disruptions to our service, and our commitment to innovation remains stronger than ever.
```
Comment (bradystroud) [inline content/blog/version-2.mdx:16]:
```suggestion
We were very proud to announce that TinaCMS had officially become part of the SSW family. Acquired by SSW, an Australian software development company and long-time Tina customer, this acquisition marked a new chapter for TinaCMS. This partnership will allow us to further enhance our platform and continue delivering top-notch solutions to our community. Rest assured, there will be no disruptions to our service, and our commitment to innovation remains stronger than ever.
```

Comment (bradystroud) [inline content/blog/version-2.mdx:16]:
```suggestion
We were very proud to announce that TinaCMS had officially become part of the SSW family. Acquired by SSW, an Australian software development company and long-time Tina customer, this acquisition marked a new chapter for TinaCMS. This partnership will allow us to further enhance our platform and continue delivering top-notch solutions to our community. Rest assured, there will be no disruptions to our service, and our commitment to innovation remains stronger than ever.
```

Comment (bradystroud) [inline content/blog/version-2.mdx:16]:
```suggestion
We were very proud to announce that TinaCMS had officially become part of the SSW family. Acquired by SSW, an Australian software development company and long-time Tina customer, this acquisition marked a new chapter for TinaCMS. This partnership will allow us to further enhance our platform and continue delivering top-notch solutions to our community. Rest assured, there will be no disruptions to our service, and our commitment to innovation remains stronger than ever.
```

Comment (bradystroud) [inline content/blog/version-2.mdx:16]:
```suggestion
We were very proud to announce that TinaCMS had officially become part of the SSW family. Acquired by SSW, an Australian software development and long-time Tina customer, this acquisition marked a new chapter for TinaCMS. This partnership will allow us to further enhance our platform and continue delivering top-notch solutions to our community. Rest assured, there will be no disruptions to our service, and our commitment to innovation remains stronger than ever.
```

**Review by bradystroud — CHANGES_REQUESTED** [review]

File: content/blog/version-2.mdx (hunk 2 above)
Comment (bradystroud) [inline content/blog/version-2.mdx:16]:
```suggestion
We were very proud to announce that TinaCMS had officially become part of the SSW family. Acquired by SSW, an Australian software consultancy and long-time Tina customer, this acquisition marked a new chapter for TinaCMS. This partnership will allow us to further enhance our platform and continue delivering top-notch solutions to our community. Rest assured, there will be no disruptions to our service, and our commitment to innovation remains stronger than ever.
```

**Review by bradystroud — CHANGES_REQUESTED** [review]

File: content/blog/version-2.mdx (hunk 2 above)
Comment (bradystroud) [inline content/blog/version-2.mdx:16]:
```suggestion
We were very proud to announce that TinaCMS had officially become part of the SSW family. Acquired by SSW, an Australian software development company and long-time Tina customer, this acquisition marked a new chapter for TinaCMS. This partnership will allow us to further enhance our platform and continue delivering top-notch solutions to our community. Rest assured, there will be no disruptions to our service, and our commitment to innovation remains stronger than ever.
```

**Review by bradystroud — CHANGES_REQUESTED** [review]

File: content/blog/version-2.mdx (hunk 2 above)
Comment (bradystroud) [inline content/blog/version-2.mdx:16]:
```suggestion
We were very proud to announce that TinaCMS had officially become part of the SSW family. Acquired by SSW, an Australian software development company and long-time Tina customer, this acquisition marked a new chapter for TinaCMS. This partnership will allow us to further enhance our platform and continue delivering top-notch solutions to our community. Rest assured, there will be no disruptions to our service, and our commitment to innovation remains stronger than ever.
```

Comment (bradystroud) [inline content/blog/version-2.mdx:16]:
```suggestion
We were very proud to announce that TinaCMS had officially become part of the SSW family. Acquired by SSW, an Australian software development company and long-time Tina customer, this acquisition marked a new chapter for TinaCMS. This partnership will allow us to further enhance our platform and continue delivering top-notch solutions to our community. Rest assured, there will be no disruptions to our service, and our commitment to innovation remains stronger than ever.
```

**Review by landonmaxwell — APPROVED** [review]
lgtm

**Comment by bradystroud** [comment]
As per my conversation with @JackDevAU waiting for approval from some more people

**Comment by bradystroud** [comment]
Merging because its been open for months, it probably wont have that much traffic and its easy to change if there are bugs.

**Pull request merged**